src/helpers/__mocks__/articleResponses.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Can you make this file more DRY, have a base definition for your article, likes and dislikes, then let other objects consume this using your spread operator